src/dex-ui/pages/dao/DAOProposals/Forms/DAOTokenTransferDetailsForm.tsx

```diff
@@ -86,18 +92,6 @@ export function DAOTokenTransferDetailsForm() {
       };
     }) ?? [];
 
-  useEffect(() => {
-    if (state?.tokenId) {
-      setValue("tokenId", state?.tokenId);
-    }
-  }, [setValue, state?.tokenId]);
-
```

Comment:
When we are navigating from Assets to Token transfer, selected asset is not getting selected in dropdown in token transfer. are we removing this feature?